tests/test_glm_synthetic.py

```diff
@@ -49,26 +49,26 @@
 # ))
 fitted_model.fit(spike_data)
 fit_pred = fitted_model.predict(spike_data)
+#
+# fig, axes = plt.subplots(2, 1)
+# axes[0].plot(onp.arange(nt), spike_data[0])
+# axes[0].plot(onp.arange(ws, nt + 1), sim_pred[0])
+# axes[0].plot(onp.arange(ws, nt + 1), fit_pred[0])
+# axes[1].plot(onp.arange(nt), spike_data[1])
+# axes[1].plot(onp.arange(ws, nt + 1), sim_pred[1])
+# axes[1].plot(onp.arange(ws, nt + 1), fit_pred[1])
+# plt.show()
```

Comment:
why are these lines commented out?

Reply:
all this test code will need to be re-written. I'll move visualization scripts into the scripts we will use for the documentation

Reply:
reverted tests to the main version. tests will be reformatted when the functions that they are pointing to will be modified